scripts/bootstrap/compile.sh

```diff
@@ -207,10 +207,24 @@ if [ -z "${BAZEL_SKIP_JAVA_COMPILATION}" ]; then
     cp src/main/java/$i ${OUTPUT_DIR}/classes/$i
   done
 
+  # Create the bazel_tools repository.
+  BAZEL_TOOLS_REPO=${OUTPUT_DIR}/embedded_tools
+  mkdir -p ${BAZEL_TOOLS_REPO}
+  cat <<EOF >${BAZEL_TOOLS_REPO}/WORKSPACE
+worksapce(name = 'bazel_tools')
+EOF
+  link_dir `pwd`/src ${BAZEL_TOOLS_REPO}/src
```

Comment:
nit: use $PWD instead of \`pwd\` to save creating a few subprocesses and make the script a bit faster on Windows

Reply:
Done.